bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableBufferedMutatorHelper.java

```diff
@@ -78,16 +78,18 @@ public class BigtableBufferedMutatorHelper {
   public BigtableBufferedMutatorHelper(
       BigtableApi bigtableApi, BigtableHBaseSettings settings, HBaseRequestAdapter adapter) {
     this.adapter = adapter;
-    this.settings = settings;
+    this.settings =
+        settings; // this settings should have all the configs from CloudBigtableTableConfiguration
```

Comment:
I think we should remove this comment here. CloudBigtableTableConfiguration is only relevant to dataflow. Adding the comment here feels like we only use bufferedMutator for dataflow which is not true.

Reply:
ok. But just curious what else this is designed for?

Reply:
It implements hbase buffered mutator: https://hbase.apache.org/apidocs/org/apache/hadoop/hbase/client/BufferedMutator.html for batching requests, which are not always coming from dataflow

Reply:
oh ok, so in hbase this is a separate class for batch mutation? Like it's not part of the regular hbase client?
Why do people have to create separate things for it instead of just adding the function to the client?

Reply:
Sorry, not sure I follow. BufferedMutator is the hbase equivalent of our batcher

Reply:
no worries I just wondered why the batcher needs to be a separate class, why not adding the batching logic to the client directly.